src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -688,9 +688,30 @@ private function addTranslatorSection(ArrayNodeDefinition $rootNode)
                             ->prototype('scalar')->end()
                         ->end()
                         ->arrayNode('enabled_locales')
-                            ->prototype('scalar')
+                            ->prototype('scalar')->end()
                             ->defaultValue([])
                         ->end()
+                        ->arrayNode('pseudo_localization')
+                            ->canBeEnabled()
+                            ->children()
+                                ->booleanNode('accents')->defaultTrue()->end()
+                                ->floatNode('expansion_factor')
+                                    ->validate()
+                                        ->ifTrue(static function (float $v): bool {
+                                            return $v < 1.0;
+                                        })
+                                        ->thenInvalid('The "expansion_factor" option must be greater than or equal to 1.')
+                                    ->end()
+                                    ->defaultValue(1.0)
+                                ->end()
+                                ->booleanNode('brackets')->defaultTrue()->end()
+                                ->booleanNode('parse_html')->defaultFalse()->end()
+                                ->arrayNode('localizable_html_attributes')
```

Comment:
missing `->fixXmlConfig` on the `pseudo_localization` to properly support XML with a singular tag name.